images/linux/scripts/installers/vercel.sh

```diff
@@ -1,31 +1,31 @@
 #!/bin/bash
 ################################################################################
-##  File:  zeit-now.sh
-##  Desc:  Installs the Zeit Now CLI
+##  File:  vercel.sh
+##  Desc:  Installs the Vercel CLI
 ################################################################################
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-# Install the Zeit Vercel CLI
+# Install the Vercel CLI
 npm i -g vercel
 
 # Validate the installation
 echo "Validate the installation"
```

Comment:
Perhaps, it's make sense to move this block below symlink creation, to separate installation from validation.

Reply:
IMHO In this case, if Vercel installation will be failed we can get additional error in symlink creation.
I think, better to check Vercel installation before symlink creation as it already done